Zilon.Core/Zilon.Core/MapGenerators/StaticObjectFactories/TrashHeapFactory.cs

```diff
@@ -10,5 +11,8 @@ public sealed class TrashHeapFactory : PropDepositFactoryBase
             IDropResolver dropResolver) : base(toolTags: new[] { "shovel" }, dropTableSchemeSid: "trash-heap", PropContainerPurpose.TrashHeap, schemeService, dropResolver)
         {
         }
+
+        protected override int ExhausingValue { get => 3; }
+        protected override DepositMiningDifficulty DepositMiningDifficulty { get => DepositMiningDifficulty.Easy; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a way to break out of this property accessor's recursion.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5398720)